Rdmp.Core/CommandLine/Interactive/ConsoleInputManager.cs

```diff
@@ -272,9 +273,6 @@ public override DirectoryInfo SelectDirectory(string prompt)
 
         public override FileInfo SelectFile(string prompt)
         {
-            if (DisallowInput)
```

Comment:
Lets keep this in, method is virtual and public so could be overridden.  Better safe than sorry.